bitnami/spark/templates/configmap.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
```

Comment:
If you use the configurationConfigMap settings this should not be created

Reply:
Done.